src/main/java/com/amazonaws/services/sqs/AmazonSQSRequesterClientBuilder.java

```diff
@@ -17,6 +17,7 @@
     private int idleQueueSweepingPeriod = 5;
     private long queueHeartbeatInterval = AmazonSQSIdleQueueDeletingClient.HEARTBEAT_INTERVAL_SECONDS_DEFAULT;
     private TimeUnit idleQueueSweepingTimeUnit = TimeUnit.MINUTES;
+    private long queueRetentionPeriodSeconds = AmazonSQSTemporaryQueuesClientBuilder.QUEUE_RETENTION_PERIOD_SECONDS_DEFAULT;
```

Comment:
Apologies for not spotting this earlier, but this should really be *idle*QueueRetentionPeriodSeconds in all these new places. The distinction has caused confusion for customers in the past. You can't change the existing public constants, but at least new references can be more precise.

Reply:
updated queueRetentionPeriodSeconds to idleQueueRetentionPeriodSeconds in all the new places